go.mod

```diff
@@ -25,17 +25,19 @@ require (
 	golang.org/x/crypto v0.6.0
 	gopkg.in/inf.v0 v0.9.1
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.26.4
-	k8s.io/apiextensions-apiserver v0.26.4
-	k8s.io/apimachinery v0.26.4
-	k8s.io/client-go v0.26.4
-	k8s.io/kube-aggregator v0.26.4
+	k8s.io/api v0.26.5
+	k8s.io/apiextensions-apiserver v0.26.5
+	k8s.io/apimachinery v0.26.5
+	k8s.io/client-go v0.26.5
+	k8s.io/kube-aggregator v0.26.5
 	k8s.io/utils v0.0.0-20230313181309-38a27ef9d749
 	sigs.k8s.io/controller-runtime v0.14.6
 	sigs.k8s.io/kind v0.17.0 // Do not remove, not used by code but used by build
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require k8s.io/apiserver v0.26.5
```

Comment:
I don't know why we have 2 separate require blocks in this file but do we need a 3rd one with this on its own?

Reply:
This was an auto-generated change from `go mod tidy`, not sure why it did this

Reply:
I'll try putting it with the others.

Reply:
Fixed